tools/read-version

```diff
@@ -133,8 +119,6 @@ data = {
 if output_json:
     sys.stdout.write(json.dumps(data, indent=1) + "\n")
 else:
-    sys.stdout.write(release + "\n")
+    sys.stdout.write(version + "\n")
```

Comment:
I think we want to leave this as release (which will be Major.Minor) so that we don't break python sdist builds, which call tools/read-version directly as well.

Otherwise we hit the sdist wheel errors as seen in CI on this branch


```
  WARNING: Built wheel for cloud-init is invalid: Metadata 1.2 mandates PEP 440 version, but '22.4-16-g4f7e8b9a' is not
```

Reply:
I pushed another possibility for for the sdist builds to replace the first '-' with a '+'. Do you think that would work?